com/course/zenkouv/src/Lecture3/Task1/Knight.java

```diff
@@ -0,0 +1,124 @@
+package Lecture3.Task1;
+
+import Lecture3.Task1.comparator.CostComparator;
+import Lecture3.Task1.comparator.WeightComparator;
+import Lecture3.Task1.model.Armor;
+import Lecture3.Task1.model.Outfit;
+import Lecture3.Task1.model.Weapon;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Knight {
+
+    private final String name;
+    private final List<Outfit> outfitList;
+    private int outfitsCost;
+
+    public Knight(String name) {
+        this.name = name;
+        this.outfitList = new ArrayList<>();
```

Comment:
move Array list initialization to init field

Reply:
fix